tools/bioimaging/bioimage_inference.xml

```diff
@@ -0,0 +1,86 @@
+<tool id="bioimage_inference" name="Process image using a BioImage.IO model" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@">
+    <description>with PyTorch</description>
+    <macros>
+        <token name="@TOOL_VERSION@">2.3.1</token>
+        <token name="@VERSION_SUFFIX@">0</token>
+    </macros>
+    <creator>
+	<organization name="BioImage.IO" url="https://bioimage.io/#/" email="" />
+	<person name="Anup Kumar" email="kumara@informatik.uni-freiburg.de" />
+	<person name="Beatriz Serrano-Solano" email="" />
+	<person name="Leonid Kostrykin" email="" />
+    </creator>
+    <edam_operations>
+        <edam_operation>operation_3443</edam_operation>
+    </edam_operations>
+    <xrefs>
+        <xref type="bio.tools">pytorch</xref>
+        <xref type="biii">pytorch</xref>
+    </xrefs>
+    <requirements>
+        <requirement type="package" version="3.9.12">python</requirement>
+        <requirement type="package" version="2.3.1">pytorch</requirement>
+	    <requirement type="package" version="0.18.1">torchvision</requirement>
+	    <requirement type="package" version="2.34.2">imageio</requirement>
+    </requirements>
+    <version_command>echo "@VERSION@"</version_command>
+    <command detect_errors="aggressive">
+    <![CDATA[
+        python '$__tool_directory__/main.py'
+
+            --imaging_model '$input_imaging_model'
+            --image_file '$input_image_file'
+	        --image_size '$input_image_input_size'
+
+    ]]>
+    </command>
+    <inputs>
+        <param name="input_imaging_model" type="data" format="zip" label="BioImage.IO model" help="Please upload a BioImage.IO model."/>
+	<param name="input_image_file" type="data" format="tiff,png" label="Input image" help="Please provide an input image for the analysis."/>
+	<param name="input_image_input_size" type="text" label="Shape of the input image" help="Provide shape of input image. See chosen model's RDF file to find correct input shape. For example: for the BioImage.IO model MitochondriaEMSegmentationBoundaryModel, the input shape is 256 x 256 x 32 x 1. Enter the shape as '256,256,32,1'."/>
```

Comment:
Ok, I replaced the `x` symbol by `*`. For this sentence `Enter the shape as '256,256,32,1'`, I have left out any quotation or symbols to avoid confusion. It should be entered just like any other text.

Reply:
@anuprulez I just looked into the RDF file because I thought it would be good to use their notation and saw that they use the `x` notation :) much sorry! I didn't know that you sticked to the RDF notation… so actually using the `x` notation which you used originally, then is the best option, I think.